app/assets/v2/images/hackathon/ethhack/banner.svg

```diff
@@ -0,0 +1,9 @@
+
+<svg width="3009px" height="609px" viewBox="0 0 3009 609" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <g id="Page-2" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <rect id="Rectangle" fill="#180944" fill-rule="nonzero" x="0" y="0" width="3009" height="609"></rect>
+        <g id="image-1440x480" transform="translate(1346.000000, 140.000000)">
```

Comment:
we have this image without being base64 here https://github.com/gitcoinco/web/blob/master/app/assets/v2/images/hackathon/ethereal.svg#L1